server/log_rpc_server_test.go

```diff
@@ -1135,6 +1133,7 @@ func TestGetSequencedLeafCount(t *testing.T) {
 type consistProofTest struct {
 	req         trillian.GetConsistencyProofRequest
 	wantErr     bool
+	errStr      string
```

Comment:
Could just use `errStr!=""` to indicate `wantErr`

Reply:
Done.